README.md

```diff
@@ -111,6 +112,9 @@ _Note: Links are provided further down to create an app for your logged in user
 **Organization Permissions**
 * Self-hosted runners (read / write)
 
+**Subscribe to events**
+* Check run (if you are going to use [Faster Autoscaling with GitHub Webhook](#faster-autoscaling-with-github-webhook))
```

Comment:
Ah so you need to configure a subscription to let your github apps' webhook to receive check run events? (I have not yet tested it with GitHub App myself so just confirming

Reply:
I haven't managed to setup the App in other way. From the very beginning I hadn't any App subsriptions, autoscaling wasn't working